setup.cfg

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=test_settings
-addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8 --cov --nomigrations
+addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8
```

Comment:
I would probably normally agree but the tests are pretty slow even running locally and any way that they can be sped up...